src/composables/useTable.ts

```diff
@@ -165,11 +171,101 @@ export const useTable = () => {
     }
   };
 
+  const sortByTeam = (
+    rows: readonly TableRow[],
+    sortBy: string,
+    descending: boolean,
+  ): readonly TableRow[] => {
+    const data = [...rows];
+
+    if (sortBy) {
+      data.sort((a, b) => {
+        const aVal = a[sortBy] ? a[sortBy] : '';
+        const bVal = b[sortBy] ? b[sortBy] : '';
+        if (!aVal || !bVal) return 0;
```

Comment:
Why not use `if else if else` condition statement or `switch` statement

```js
if (...) {
} else if (...) {
} else { 
}
```

?

Reply:
Updated using the if - else if structure.